src/running-the-course/translations.md

```diff
@@ -25,6 +26,7 @@ get going. Translations are coordinated on the [issue tracker].
 [ja]: https://google.github.io/comprehensive-rust/ja/
 [ko]: https://google.github.io/comprehensive-rust/ko/
 [pt-BR]: https://google.github.io/comprehensive-rust/pt-BR/
+[bn]: https://google.github.io/comprehensive-rust/bn/
```

Comment:
Can't make a suggested change here, but again, make them sorted (bn should come first)